swan-lake/learn/testing-ballerina-code/writing-tests.md

```diff
@@ -318,9 +318,48 @@ function testFunction2() {
 }
 ```
 
-### @test:BeforeEach {}
+### @test:BeforeGroups {}
 
-The function specified following the annotation will be run before every test within the test suite is run. This can be used for repeatedly initializing test-level aspects before every test function.
+The function specified following the annotation will be run will execute once before the first test belonging to the
```

Comment:
```suggestion
Once the function specified after the annotation will be run, it will execute once before the first test belonging to the
```
The meaning is not clear.

Reply:
Addressed in af5c8a9